tests/unit-core/test-api.c

```diff
@@ -145,7 +145,10 @@ handler_construct (const jerry_value_t func_obj_val, /**< function object */
   TEST_ASSERT (jerry_get_boolean_value (args_p[0]) == true);
 
   jerry_value_t field_name = jerry_create_string ((jerry_char_t *) "value_field");
-  jerry_set_property (this_val, field_name, args_p[0]);
+  jerry_value_t res = jerry_set_property (this_val, field_name, args_p[0]);
+  TEST_ASSERT (!jerry_value_is_error (res));
+  TEST_ASSERT (jerry_get_boolean_value (res));
```

Comment:
Please add a `jerry_get_boolean_value` call too to ensure the result was a success. 
```c
TEST_ASSERT (jerry_get_boolean_value (res) && jerry_get_boolean_value (res));
```